.DoxygenLayout.xml

```diff
@@ -21,174 +21,173 @@
     </tab>
     <tab type="examples" visible="yes" title="Examples" intro=""/> 
   </navindex>
+	<!-- Layout definition for a class page -->
```

Comment:
I suspect these modifications to .DoxygenLayout.xml are a result of a "git add *". Can you restore this file from dev/master . If you meant to update the layout I'd rather it be done as a separate PR since it is unrelated to offline tracers.